capstone/client/src/components/StoresPage.js

```diff
@@ -14,64 +14,59 @@
  * limitations under the License.
  */
 
-import React, { Component } from 'react';
 import axios from 'axios';
-import { Grid, Card} from '@material-ui/core';
-import { GoogleApiWrapper } from 'google-maps-react';
+import { Card, Grid} from '@material-ui/core';
+import React, { Component } from 'react';
 
 import { Store } from './Store';
-import APIKey from './APIKey.js';
-import StoreOverviewCards from './StoreOverviewCards.js';
 import StoreDetailCards from './StoreDetailCards.js';
-import './styles.css';
+import StoreOverviewCards from './StoreOverviewCards.js';
 
+import './styles.css';
 
 const itemList = [
     "Milk",
     "Bread"
 ]
 
 class StorePage extends Component { 
-    constructor(props) {
-      super(props)
-      this.state = {
-          stores : []
-      }
-      this.getStores = this.getStores.bind(this);
+  constructor(props) {
+    super(props)
+    this.state = {
+      stores : []
     }
+    this.getStores = this.getStores.bind(this);
```

Comment:
Move this line to componentDidMount, that should fix the error

Reply:
Hm, the error is still there.